packages/flutter_tools/pubspec.yaml

```diff
@@ -11,7 +11,7 @@ dependencies:
   # To update these, use "flutter update-packages --force-upgrade".
   archive: 2.0.11
   args: 1.5.2
-  dwds: 3.0.0
+  dwds: 3.0.2
```

Comment:
To get the analysis step to accept this, first pull upstream (I just landed a big versions update), then run `flutter update-packages --force-upgrade`

Reply:
done, thanks!

Reply:
done, thanks!